opbeans/src/main/resources/application.properties

```diff
@@ -0,0 +1,14 @@
+spring.jpa.database=POSTGRESQL
```

Comment:
What about using an in-memory database like H2?

Reply:
Yes, I need to do that part this is mainly focused for my local testing. I'm changing it to in memory db by default.